src/arrayMethodSort.js

```diff
@@ -5,7 +5,44 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let callback = compareFunction;
+    let prev;
+    let curr;
+    let count;
+
+    const defaultFunction = function(a, b) {
```

Comment:
I think not good naming :)

Reply:
Thanks, I will try to code my code more understandable, especially in variable naming ))) I see that I have some trouble with it )))